css/pages/navbar.css

```diff
@@ -21,6 +21,10 @@ nav {
 #login-btn {
   float: right;
```

Comment:
Are you sure if we need `float: right` and `text-transform: capitalize`?

There's no difference even if disable them:

![image](https://user-images.githubusercontent.com/56217868/96019395-1b2a2080-0e6a-11eb-8dbe-b1af72b68db7.png)

Reply:
Yes! Not actually. Thanks!